.github/workflows/main.yml

```diff
@@ -0,0 +1,102 @@
+name: Main
+
+on:
+  push:
+    branches:
+      # TODO this could also be run when pushing to main, but could end up clogging the CI when merging multiple PRs
+      - rel_*
+  schedule:
+    # run daily at 5:00 am UTC (12 am ET/9 pm PT)
+    - cron: '0 5 * * *'
+  repository_dispatch:
+    # to run this, send a POST API call at repos/IDAES/idaes-pse/dispatches with the specified event_type
+    # e.g. `gh repos/IDAES/idaes-pse/dispatches -F event_type=run_tests`
+    types: [run_tests]
+  workflow_dispatch:
+    inputs:
+      git-ref:
+        description: Git hash (optional)
+        required: false
+
+jobs:
+  pytest:
+    name: All tests (py${{ matrix.python-version }}/${{ matrix.os }})
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version:
+          - '3.6'
+          - '3.7'
```

Comment:
Good point. Personally I think it'd be good to include 3.8. Let's wait until tomorrow to see if there are other thoughts on this, otherwise I'll add it.

Reply:
I agree, let's add 3.8!

Reply:
:heavy_check_mark: